sku.0/sys.server/compiled/game/script/gm/cmd.java

```diff
@@ -4439,6 +4439,38 @@ else if (command.equalsIgnoreCase("setWeather")) {
             return SCRIPT_CONTINUE;
         }
 
+        else if (command.equalsIgnoreCase("setCount")) {
+            
+            int count;
+            if(st.hasMoreTokens()) {
+                count = utils.stringToInt(st.nextToken());
+            } else {
+                sendSystemMessageTestingOnly(self, "Syntax: /admin setCount <integer> ");
+                return SCRIPT_CONTINUE;
+            }
+            if (count > 500) {
+                count = 500;
+            }
+            setCount(target, count);
+            sendConsoleMessage(self, "You have added " + count + " to " + target + " (" + getName(target) + ")");
+            return SCRIPT_CONTINUE;
+        }
+
+        else if (command.equalsIgnoreCase("messageTo")){
+            String message;
+            float messageDelay;
+            if(st.hasMoreTokens()) {
+                message = st.nextToken();
+                messageDelay = utils.stringToFloat(st.nextToken());

```

Comment:
With StringTokenizer you need to check hasMoreTokens() for each parameter. This currently has the potential to throw a null pointer exception because you’re calling nextToken() twice without knowing there are actually 2 more tokens (vs just 1 for the message param).

As an aside, these command scripts could really be cleaned up with a helper method and class used to validate params and return syntax because it’s so messy the way we currently do it over and over. A todo for another time.